core/src/test/scala/org/apache/spark/deploy/history/EventLogTestHelper.scala

```diff
@@ -56,7 +56,7 @@ object EventLogTestHelper {
       eventStr: String,
       desiredSize: Long): Seq[String] = {
     val stringLen = eventStr.getBytes(StandardCharsets.UTF_8).length
-    val repeatCount = Math.floor(desiredSize / stringLen).toInt
+    val repeatCount = Math.floor(desiredSize.toDouble / stringLen).toInt
```

Comment:
This is the third case related to `Math.floor`. 
`desiredSize / stringLen` is also a long value. Is it necessary to perform `Math.floor` here?

 @HeartSaVioR  Do you have time to help check this logic? Thanks

Reply:
Done!